<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBaseUtilDlgUI</class>
 <widget class="QWidget" name="DBaseUtilDlgUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="parametersLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="runPlannerButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>run planner</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runPlannerOnCGDBButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>run on CGDB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="takeScansButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>take scans</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="handNameLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
